src/poseidon_reference.js

```diff
@@ -1,21 +1,21 @@
 
 import assert from "assert";
-import { getCurveFromName }  from "ffjavascript";
+import { getCurveFromName, Scalar, F1Field } from "ffjavascript";
 
 import poseidonConstants from "./poseidon_constants.js";
 
 function unsringifyConstants(Fr, o) {
```

Comment:
1. shall we remove `Fr` as argument since it's not used in the function?
2. why replacing `Fr` with `Scalar`?